libraries/core-react/src/components/Typography/Typography.tokens.ts

```diff
@@ -51,11 +52,26 @@ type ColorVariants =
 
 type TypographyGroups = keyof typeof typography
 
+const link: ComponentToken = {
+  states: {
+    focus: {
+      outline: {
+        width: '1px',
+        color: focus,
+        style: 'dashed',
+        type: 'outline',
+        offset: '6px',
```

Comment:
Should the offset be `6px`?

Reply:
Tbh I just copied this from the Breadcrumb component. Looking for the Typography component in Figma turned out to be harder than expected. Do you know where it is? 🤔